drivers/overwrite_queue.go

```diff
@@ -0,0 +1,119 @@
+package drivers
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/common"
+)
+
+const (
+	timeoutMultiplier = 1.5
+)
+
+type (
+	OvewriteQueue struct {
+		desc           string
```

Comment:
Should this be a manifestID instead so that the manifestID can always be included in the save data logs?

Reply:
Added manifestID to the description